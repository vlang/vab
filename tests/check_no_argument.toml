execute = 'vab'
[expect]
	exit_code = 1
