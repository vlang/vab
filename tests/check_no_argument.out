No arguments given
Use `vab -h` to see all flags